### PR TITLE
I totally know how github works shutup I know what I'm doing (i think)

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

**Review by AMiller42 — APPROVED** [review]
gotta hide dem flagz from da wolfs

**Review by PyGamer0 — APPROVED** [review]

File: hooks/msghooks.py [hunk 1]
```diff
@@ -18,6 +18,9 @@ def on_join():
         save()
         time.sleep(5)
         return "@" + data["data"]["user_name"].replace(" ", "") + " " + WELCOME
+    elif user == 347075:
```
Comment (cgccuser) [inline hooks/msghooks.py:21]: Magic numbers are no bueno, could you add a set `FLAG_HATER_BLACKLIST` containing Redwolf's id and then check for `user in FLAG_HATER_BLACKLIST`?

File: hooks/msghooks.py [hunk 2]
```diff
@@ -126,7 +129,7 @@ def receive_message():
                 else:
                     return f"{reply} No, you sussy baka."
             else:
-                if random.random() <= 0.5 or message["user_id"] == 513493:
+                if random.random() <= 0.5 or message["user_id"] == 513493 or message["user_id"] == 354515:
```
Comment (cgccuser) [inline hooks/msghooks.py:132]: Again, I don't like the magic numbers or the repetition of `message[userid] ==`, I think you should use something like `in FLAG_HATER_LIST` instead.

**Review by cgccuser — CHANGES_REQUESTED** [review]

**Comment by hyper-neutrino** [comment]
no

**Comment by Lyxal** [comment]
Then I'll have to do it myself

**Pull request closed without merging**